contracts/v2/lib/DepositContractBase.sol

```diff
@@ -89,6 +94,26 @@ contract DepositContractBase {
         assert(false);
     }
 
+    /**
+     * @notice Rollback the merkle tree to a previous state
+     * @param newDepositCount New deposit count (leaf count) to rollback to
+     * @param newFrontier New frontiers (branch array)
+     */
+    function _rollbackTree(
+        uint256 newDepositCount,
+        bytes32[_DEPOSIT_CONTRACT_TREE_DEPTH] calldata newFrontier
+    ) internal {
+        if (newDepositCount >= depositCount) {
```

Comment:
i think we can let set whathever depositCount.

Reply:
> I think we can let set whathever depositCount.

agree on this. We are limiting the functionality with this check and we may miss scenarios on which is required to set any `depositCount`. For example: The SC is created, few bridges are done, certificate is created and settled on L1....then, Sc tx is reorged. To solve the previous situation, we would need to deploy again the SC and set a depositCount > currentOne (0)

Reply:
I read it before, but then the naming of this function should be `resetTree`